packages/components/src/DataList/components/DataListLayoutInternal/__tests__/DataListHeaderCheckbox.test.tsx

```diff
@@ -112,6 +114,35 @@ describe("DataListHeaderCheckbox", () => {
   });
 
   describe("Select All Checkbox Indeterminate and Checked State", () => {
+    Object.defineProperty(window, "matchMedia", {
```

Comment:
I believe you have to reset this back to the original. Otherwise it'll be overwritten on other tests files as well.